pydy/viz/scene.py

```diff
@@ -465,59 +480,105 @@ def display(self):
     def display_ipython(self):
         """
         Method to display the visualization inside the
-        Ipython notebook. It is only supported by IPython
+        IPython notebook. It is only supported by IPython
         versions>=2.0.0
 
         """
 
+        # Raise error whenever display_ipython() is called
+        # and IPython is not installed or IPython < '2.0.0'
+        if IPython is None:
+            raise ImportError('IPython is not installed but is required. ' +
+                              'Please installed IPython >= 2.0 and try again')
+        elif ipython_less_than_3 is None:
```

Comment:
But maybe the two different error messages are nice for the user. This probably doesn't matter either way.